File: app/src/main/java/com/gigigo/ruialonso/roompersistance/db/entities/UserEntity.java
package com.gigigo.ruialonso.roompersistance.db.entities;

import android.arch.persistence.room.Entity;
import android.arch.persistence.room.PrimaryKey;

@Entity(tableName = "User")
public class UserEntity {
  @PrimaryKey public int id;

  public String name;

  public String email;
}

File: app/src/main/java/com/gigigo/ruialonso/roompersistance/db/entities/UserEntity.kt
package com.gigigo.ruialonso.roompersistance.db.entities

import android.arch.persistence.room.Entity
import android.arch.persistence.room.PrimaryKey

@Entity(tableName = "User")
class UserEntity {
  @PrimaryKey
  var id: Int = 0

  var name: String? = null

  var email: String? = null
}
